pkgs/tools/audio/acousticbrainz-client/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, fetchFromGitHub, python3Packages, essentia-extractor }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "acousticbrainz-client";
+  version = "0.1";
+
+  src = fetchFromGitHub {
+    owner = "MTG";
+    repo = "acousticbrainz-client";
+    rev = version;
+    sha256 = "1g1nxh58939vysfxplrgdz366dlqnic05pkzbqh75m79brg4yrv1";
+  };
+
+  propagatedBuildInputs = [ essentia-extractor python3Packages.requests ];
+
+  postPatch = ''
+    # Avoid installing a copy of the streaming_extractor_music binary,
+    # which is provided by essentia-extractor
+    sed -i -e "/streaming_extractor_music/d" setup.py
```

Comment:
Can we use substituteInPlace here?

Reply:
I'm not sure, the command removes the whole line where `streaming_extractor_music` appears.

Reply:
I took a different approach that does not require patching the script.

Reply:
> I'm not sure, the command removes the whole line where streaming_extractor_music appears.

Right, it cannot do that.